src/test/java/bio/terra/service/resourcemanagement/google/BucketResourceTest.java

```diff
@@ -479,7 +481,7 @@ public void createBucketWithTtlTest() throws Exception {
 
   @Test
   public void testCreateBucketWithReaders() throws Exception {
-    String bucketName = "bucket_with_reader_groups";
+    String bucketName = createBucketName("bucketwithreadergroupstest");
```

Comment:
Most calls to `createBucketName()` are before a call to `createBucket()`. I wonder if it would be worth adding code to `createBucket()` to automatically randomize. I see two cases where the bucket name is intentionally reused.. you could probably work around that but maybe it's not worth the complexity.

Reply:
That's a good idea. I'd like it if this utility is something you get "for free".

Reply:
Ah, the cases where we reuse the bucket name are actually kind of tricky, so I'll leave this be.